portfolio/src/main/webapp/script.js

```diff
@@ -36,3 +36,18 @@ async function showSomething(){
     servletContainer.innerText = textFromResponse;
 }
 
+async function getServerStats(){
+    //fetch function that will get info from /hello
+    const serverResponse = await fetch('/my-data-url');
+
+    //parsing to json
+    const info = await serverResponse.json();
+    const infoList = document.getElementById('hello-container');
+  infoList.innerHTML = '';
+
+    infoList.appendChild(
+        createListElement('Hello: ' + infoList)
```

Comment:
Possible typo: You're adding a list element to infoList, but putting infoList into it. That seems a little strange, did you mean to put info instead of infoList?

Reply:
When I was working on this I was comparing my file to the one provided in the walkthrough but didn't finish it yet. I'll be reviewing it later since a few things have been cleared up for me now.